assets/src/block-editor/helpers/index.js

```diff
@@ -133,7 +133,8 @@ export const addAMPAttributes = ( settings, name ) => {
 			attribute: 'max-font-size',
 		};
 		settings.attributes.height = {
-			default: 'core/image' === name ? 200 : 50,
+			// Needs to be higher than the maximum font size, which defaults to MAX_FONT_SIZE
+			default: 'core/image' === name ? 200 : 80,
```

Comment:
Should we use the constant here instead then, e.g. `MAX_FONT_SIZE + ...`?

Reply:
Fixed in 0e3fbe3f0ff8d66e7a78ecc7368c733189935e9a